src/test/DssProxyActionsCharter.t.sol

```diff
@@ -433,86 +516,138 @@ contract DssProxyActionsTest is DssDeployTestBase, ProxyCalls {
         charter.file("ETH", "gate", 0);
         charter.file("ETH", "Nib", 2.0 * 1e16); // two percent
 
+        uint256 cdp = this.open("ETH", address(proxy));
         uint256 initialBalance = address(this).balance;
         assertEq(ink("ETH", charterProxy), 0);
-        assertEq(dai.balanceOf(address(this)), 0);
-        this.lockETHAndDraw{value: 2 ether}(address(jug), address(ethManagedJoin), address(daiJoin), 300 ether);
+        assertEq(dai.balanceOf(charterProxy), 0);
```

Comment:
I don't think this should have changed to `charterProxy`.
On the other hand I think we should be checking `ink`, `art`, `gem` and `dai` balances against the owner of the `cdp` instead of directly `address(this)`. I know at this stage is the same but that should probably be fixed at some point. Anyway this is something that could be changed later as well.

Reply:
Right, fixed to check dai balance of address(this) as for now it aligns with other tests.
I suggest to indeed move to check the owner only once we support delegation.
Will probably have to check both address(this) and another owner then.

Reply:
Sounds good.